app/assets/javascripts/grant_permissions.js

```diff
@@ -1,11 +1,28 @@
-// Place all the behaviors and hooks related to the matching controller here.
-// All this logic will automatically be available in application.js.
-//
 //= require select2
 $(document).ready(function() {
-  $('select#grant_permission_user_id').select2({
-    placeholder: "Find an email address",
-    minimumInputLength: 1,
-    allowClear: true
-  });
-});
\ No newline at end of file
+
+  $userSelector = $('select#grant_permission_user_id')
+
+  if ($userSelector.length) {
+    $userSelector.select2({
+      placeholder: "Find an email address",
```

Comment:
Search by email address/Lookup users by email? You are searching for a user, not email